### ISSUE TITLE
Response was error login_required

### ISSUE THREAD

**Issue by vaarenyam** [IdentityServer/IdentityServer4#2800]
I am getting "Response was error login_required"  on the browser for the app deployed on the PROD environment. However, for the app running off my laptop I do not get this error. This is making it even more confusing for me to troubleshoot! The only different between the PRD and local deployment is the SSL certificate being an authentic one on the PRD server.

I have seen the other posts pertaining to this issue. However, I could not find any concrete reason or solution mentioned in those posts for this issue.

Please advise.

**UPDATE**
I have observed the following:

When I browse the URL: https://eservices.abcd.co.uk/TelcoIdentity/account/login, I do not get the above error. However, whenever I browse https://eservices.abcd.co.uk/telcoidentity/account/login OR https://eservices.abcd.co.uk/Telcoidentity/account/login OR https://eservices.abcd.co.uk/telcoIdentity/account/login, the Login_Required error shows up in the Console.

In short, the URL with the **T**elco**I**dentity, i.e. T and I in capitals works! No other URL works if any of the casing is different!!

I recently became aware that URLs are case sensitive. How do I get around this?

Please advise.

**Comment by brockallen** [comment]
> This is making it even more confusing for me to troubleshoot!

Check your logs.

**Comment by leastprivilege** [comment]
any update?
